changelogs/unreleased/gh-7223-add-more-error-details.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/box
```

Comment:
Looks more like a feature, not a bug fix, to me.

Reply:
Decided to keep as bugfix after f2f discussion.